### PR TITLE
Branch Staging but with options in quizzes changing colors after answering question

### PR DESCRIPTION
**Pull request by arsen-ti**
https://www.loom.com/share/45220faea9a649719194a255d0215fbe
https://www.loom.com/share/2f5b39e0ce08423c88192ac28f1d7d68

### CODE REVIEW

**Comment by arsen-ti** [comment]
> Screenshots dude, screenshots of what it actually looks like when a correct or wrong answer is selected and user selects show answer

https://www.loom.com/share/45220faea9a649719194a255d0215fbe
https://www.loom.com/share/2f5b39e0ce08423c88192ac28f1d7d68
I added those in the group yesterday

File: src/components/Home/QuizSection.tsx [hunk 1]
```diff
@@ -96,29 +99,34 @@ const QuizSection: FC<QuizSectionProps> = ({ quiz, refetch, origin }) => {
...
             src={(arrayHints.current[quizIndex] == null) ? "" : arrayHints.current[quizIndex] as string}
-            alt={'Hint could not be loaded/displayed at the URL: ' + arrayHints.current[quizIndex] as string} />}
+            alt={"Hint could not be loaded/displayed at the URL: ${arrayHints.current[quizIndex]}"} />}
           <RadioGroup
             value={choice}
             onChange={handleChange}
           >
             {options?.map((op, idx) => {
-              return <FormControlLabel value={op.id} key={idx} control={<Radio />} label={op.desc} />;
+              return <FormControlLabel value={op.id} key={idx} control={<Radio />} label={op.desc} className={`bg-opacity-70 ${(choice == op.id) && (op.id != correctAnswerId.current) && attempted ? "bg-red-600" : (op.id == correctAnswerId.current) && attempted ? "bg-lime-500" : ""}`} onChange={e => { console.log("the op.id ", op.id) }
```
Comment (gao-ti) [inline src/components/Home/QuizSection.tsx:128]: remove debug msg

File: src/components/Home/QuizSection.tsx [hunk 2]
```diff
@@ -132,6 +140,7 @@ const QuizSection: FC<QuizSectionProps> = ({ quiz, refetch, origin }) => {
 
   const handleChange = (e: any) => {
     setChoice(e.target.value as string);
+    console.log("the correct id ", correctAnswerId.current);
```
Comment (gao-ti) [inline src/components/Home/QuizSection.tsx:143]: remove debug msg

File: src/components/Home/QuizSection.tsx [hunk 3]
```diff
@@ -156,6 +165,7 @@ const QuizSection: FC<QuizSectionProps> = ({ quiz, refetch, origin }) => {
       quizId: quiz[quizIndex]?.id as number,
       grade: score,
     });
+
```
Comment (gao-ti) [inline src/components/Home/QuizSection.tsx:168]: remove

File: src/components/Home/QuizSection.tsx (hunk 1 above)
Comment (gao-ti) [reply src/components/Home/QuizSection.tsx:128]: make multi line

**Pull request merged**